### PR TITLE
Merge v1.11.0-beta.1 release into dev.

### PR DESCRIPTION
**Pull request by fernandofloresg**
##### SUMMARY

As part of the release process we need to merge back the release branch into the `dev` branch.

##### ISSUE TYPE

- Release Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: tests/functional/modules/test_zos_mvs_raw_func.py [hunk 1]
```diff
@@ -1301,6 +1361,7 @@ def test_write_to_existing_file(ansible_zos_module):
             ],
         )
         results2 = hosts.all.command(cmd=f"cat {DEFAULT_PATH_WITH_FILE}")
+        results2 = hosts.all.command(cmd=f"cat {DEFAULT_PATH_WITH_FILE}")
```
Comment (richp405) [inline tests/functional/modules/test_zos_mvs_raw_func.py:1364]: duplicate line

File: tests/functional/modules/test_zos_mvs_raw_func.py [hunk 2]
```diff
@@ -1426,6 +1487,7 @@ def test_file_path_options(ansible_zos_module, access_group, status_group):
...
```
Comment (richp405) [inline tests/functional/modules/test_zos_mvs_raw_func.py:1490]: duplicate line

**Review by richp405 — CHANGES_REQUESTED** [review]
Everything looks great, except for test_zos_mvs_raw_func.py
It looks like the merge went bad, because there were many > 20 duplicate lines.  I marked most of them, then realized I didn't want to swamp you.

File: plugins/action/zos_job_submit.py [hunk 3]
```diff
@@ -24,6 +24,11 @@
 from ansible_collections.ibm.ibm_zos_core.plugins.module_utils import template
 
 
+display = Display()
+
+from ansible_collections.ibm.ibm_zos_core.plugins.module_utils import template
```
Comment (rexemin) [inline plugins/action/zos_job_submit.py:29]: This import and the line above it are duplicates squeezed between the original code

Comment (fernandofloresg) [reply plugins/action/zos_job_submit.py]: done

File: tests/functional/modules/test_zos_mvs_raw_func.py (hunk 2 above)
Comment (fernandofloresg) [reply tests/functional/modules/test_zos_mvs_raw_func.py]: done

File: docs/source/resources/releases_maintenance.rst [hunk 4]
```diff
@@ -89,6 +89,11 @@ The z/OS managed node includes several shells, currently the only supported shel
 +---------+----------------------------+---------------------------------------------------+---------------+---------------+
 | Version | Controller                 | Managed Node                                      | GA            | End of Life   |
 +=========+============================+===================================================+===============+===============+
+| 1.11.x  |- `ansible-core`_ >=2.15.x  |- `z/OS`_ V2R4 - V2Rx                              | In preview    | TBD           |
```
Comment (ddimatos) [inline docs/source/resources/releases_maintenance.rst:92]: Weren't we supposed to have z/OS 3.1 support for the beta?
```
`z/OS`_ V2R4 - V3Rx
```

File: plugins/modules/zos_blockinfile.py [hunk 5]
```diff
@@ -39,7 +39,7 @@
         PS (sequential data set), member of a PDS or PDSE, PDS, PDSE.
       - The USS file must be an absolute pathname.
       - Generation data set (GDS) relative name of generation already
-        created.  C(e.g. SOME.CREATION(-1).)
+        created.  C(e.g. SOME.CREATION(-1\).)
```
Comment (ddimatos) [inline plugins/modules/zos_blockinfile.py:42]: This is a raw string , is it necessary to escape the closing paren:
```
(-1\).)
```
Could this be the source of some of the hanging closing parens we saw during doc generation.

File: plugins/modules/zos_lineinfile.py [hunk 6]
```diff
@@ -37,7 +37,7 @@
         PS (sequential data set), member of a PDS or PDSE, PDS, PDSE.
       - The USS file must be an absolute pathname.
       - Generation data set (GDS) relative name of generation already
-        created. C(e.g. SOME.CREATION(-1).)
+        created. C(e.g. SOME.CREATION(-1\).)
```
Comment (ddimatos) [inline plugins/modules/zos_lineinfile.py:40]: This is a raw string , is it necessary to escape the closing paren:
```
(-1\).)
```
Could this be the source of some of the hanging closing parens we saw during doc generation.

File: plugins/modules/zos_unarchive.py [hunk 7]
```diff
@@ -146,7 +146,7 @@
     description:
       - A list of directories, files or data set names to extract from the
         archive.
-      - GDS relative names are supported C(e.g. USER.GDG(-1)).
+      - GDS relative names are supported C(e.g. USER.GDG(-1\)).
```
Comment (ddimatos) [inline plugins/modules/zos_unarchive.py:149]: This is a raw string , is it necessary to escape the closing paren:
```
(-1\).)
```
Could this be the source of some of the hanging closing parens we saw during doc generation.

File: plugins/modules/zos_unarchive.py [hunk 8]
```diff
@@ -36,7 +36,7 @@
       - I(src) can be a USS file or MVS data set name.
       - USS file paths should be absolute paths.
       - MVS data sets supported types are C(SEQ), C(PDS), C(PDSE).
-      - GDS relative names are supported C(e.g. USER.GDG(-1)).
+      - GDS relative names are supported C(e.g. USER.GDG(-1\)).
```
Comment (ddimatos) [inline plugins/modules/zos_unarchive.py:39]: This is a raw string , is it necessary to escape the closing paren:
```
(-1\).)
```
Could this be the source of some of the hanging closing parens we saw during doc generation.

File: plugins/modules/zos_unarchive.py [hunk 9]
```diff
@@ -157,7 +157,7 @@
     description:
       - List the directory and file or data set names that you would like to
         exclude from the unarchive action.
-      - GDS relative names are supported C(e.g. USER.GDG(-1)).
+      - GDS relative names are supported C(e.g. USER.GDG(-1\)).
```
Comment (ddimatos) [inline plugins/modules/zos_unarchive.py:160]: This is a raw string , is it necessary to escape the closing paren:
```
(-1\).)
```
Could this be the source of some of the hanging closing parens we saw during doc generation.

**Review by ddimatos — CHANGES_REQUESTED** [review]
Thanks @fernandofloresg - left you a few things to review.

File: docs/source/modules/zos_copy.rst [hunk 10]
```diff
@@ -663,8 +665,11 @@ template_parameters
 
     | **required**: False
     | **type**: str
-    | **default**: \\n
-    | **choices**: \\n, \\r, \\r\\n
+    | **default**:
```
Comment (rexemin) [inline docs/source/modules/zos_copy.rst:668]: The newline options got funky again

File: docs/source/modules/zos_job_submit.rst [hunk 11]
```diff
@@ -248,8 +248,11 @@ template_parameters
 
     | **required**: False
     | **type**: str
-    | **default**: \\n
-    | **choices**: \\n, \\r, \\r\\n
+    | **default**:
```
Comment (rexemin) [inline docs/source/modules/zos_job_submit.rst:251]: Newline options also got scrambled here

File: docs/source/modules/zos_script.rst [hunk 12]
```diff
@@ -220,8 +220,11 @@ template_parameters
 
     | **required**: False
     | **type**: str
-    | **default**: \\n
-    | **choices**: \\n, \\r, \\r\\n
+    | **default**:
```
Comment (rexemin) [inline docs/source/modules/zos_script.rst:223]: Newline options need to also get fixed here

File: docs/source/modules/zos_backup_restore.rst [hunk 13]
```diff
@@ -48,17 +48,21 @@ data_sets
 
   include
     When *operation=backup*, specifies a list of data sets or data set patterns to include in the backup.
+    When *operation=backup*, specifies a list of data sets or data set patterns to include in the backup.
```
Comment (richp405) [inline docs/source/modules/zos_backup_restore.rst:51]: duplicate line in .rst

File: docs/source/modules/zos_backup_restore.rst [hunk 14]
```diff
@@ -48,17 +48,21 @@ data_sets
...
 
     When *operation=backup* GDS relative names are supported.
 
     When *operation=restore*, specifies a list of data sets or data set patterns to include when restoring from a backup.
 
+    The single asterisk, ``*``, is used in place of exactly one qualifier. In addition, it can be used to indicate to DFSMSdss that only part of a qualifier has been specified.
     The single asterisk, ``*``, is used in place of exactly one qualifier. In addition, it can be used to indicate to DFSMSdss that only part of a qualifier has been specified.
```
Comment (richp405) [inline docs/source/modules/zos_backup_restore.rst:58]: duplicate line

File: docs/source/modules/zos_backup_restore.rst [hunk 15]
```diff
@@ -48,17 +48,21 @@ data_sets
...
 
+    When used with other qualifiers, the double asterisk, ``**``, indicates either the nonexistence of leading, trailing, or middle qualifiers, or the fact that they play no role in the selection process.
     When used with other qualifiers, the double asterisk, ``**``, indicates either the nonexistence of leading, trailing, or middle qualifiers, or the fact that they play no role in the selection process.
```
Comment (richp405) [inline docs/source/modules/zos_backup_restore.rst:61]: duplicatre line

File: docs/source/modules/zos_backup_restore.rst [hunk 16]
```diff
@@ -48,17 +48,21 @@ data_sets
...
 
     Two asterisks are the maximum permissible in a qualifier. If there are two asterisks in a qualifier, they must be the first and last characters.
 
+    A question mark ``?`` or percent sign ``%`` matches a single character.
     A question mark ``?`` or percent sign ``%`` matches a single character.
```
Comment (richp405) [inline docs/source/modules/zos_backup_restore.rst:66]: duplicate line

**Review by richp405 — CHANGES_REQUESTED** [review]
one document had duplicate lines (same merge problem I saw in other files yesterday).  I left a few comments... it seems like it is only 1 file now.

**Review by rexemin — APPROVED** [review]
Thanks for working so much on this, I imagine this must have been so confusing with the merge issues. You resolved my comments and I think it's good now

File: docs/source/resources/releases_maintenance.rst (hunk 4 above)
Comment (fernandofloresg) [reply docs/source/resources/releases_maintenance.rst]: you're right, fixed

File: plugins/modules/zos_blockinfile.py (hunk 5 above)
Comment (fernandofloresg) [reply plugins/modules/zos_blockinfile.py]: fixed

File: plugins/modules/zos_lineinfile.py (hunk 6 above)
Comment (fernandofloresg) [reply plugins/modules/zos_lineinfile.py]: fixed

File: plugins/modules/zos_unarchive.py (hunk 7 above)
Comment (fernandofloresg) [reply plugins/modules/zos_unarchive.py]: fixed

File: plugins/modules/zos_unarchive.py (hunk 8 above)
Comment (fernandofloresg) [reply plugins/modules/zos_unarchive.py]: fixed

File: plugins/modules/zos_unarchive.py (hunk 9 above)
Comment (fernandofloresg) [reply plugins/modules/zos_unarchive.py]: fixed

File: docs/source/modules/zos_copy.rst (hunk 10 above)
Comment (fernandofloresg) [reply docs/source/modules/zos_copy.rst]: fixed

File: docs/source/modules/zos_job_submit.rst (hunk 11 above)
Comment (fernandofloresg) [reply docs/source/modules/zos_job_submit.rst]: regenerated the docs and looks good now

File: docs/source/modules/zos_script.rst (hunk 12 above)
Comment (fernandofloresg) [reply docs/source/modules/zos_script.rst]: regenerated the docs and looks good now

File: docs/source/modules/zos_backup_restore.rst (hunk 13 above)
Comment (fernandofloresg) [reply docs/source/modules/zos_backup_restore.rst]: regenerated the docs and looks good now

File: docs/source/modules/zos_backup_restore.rst (hunk 14 above)
Comment (fernandofloresg) [reply docs/source/modules/zos_backup_restore.rst]: regenerated the docs and looks good now

File: docs/source/modules/zos_backup_restore.rst (hunk 15 above)
Comment (fernandofloresg) [reply docs/source/modules/zos_backup_restore.rst]: regenerated the docs and looks good now

File: docs/source/modules/zos_backup_restore.rst (hunk 16 above)
Comment (fernandofloresg) [reply docs/source/modules/zos_backup_restore.rst]: regenerated the docs and looks good now

File: docs/source/modules/zos_backup_restore.rst (hunk 13 above)
Comment (ddimatos) [reply docs/source/modules/zos_backup_restore.rst]: I suspect the correction removes a diff so the file does not show up in the PR but is in in the branch as it should be so I am resolving this. 
https://raw.githubusercontent.com/ansible-collections/ibm_zos_core/cherry-picked-1110-beta-into-dev/docs/source/modules/zos_backup_restore.rst

File: docs/source/modules/zos_backup_restore.rst (hunk 15 above)
Comment (ddimatos) [reply docs/source/modules/zos_backup_restore.rst]: I suspect the correction removes a diff so the file does not show up in the PR but is in in the branch as it should be so I am resolving this. 
https://raw.githubusercontent.com/ansible-collections/ibm_zos_core/cherry-picked-1110-beta-into-dev/docs/source/modules/zos_backup_restore.rst

File: docs/source/modules/zos_backup_restore.rst (hunk 16 above)
Comment (ddimatos) [reply docs/source/modules/zos_backup_restore.rst]: I suspect the correction removes a diff so the file does not show up in the PR but is in in the branch as it should be so I am resolving this.
https://raw.githubusercontent.com/ansible-collections/ibm_zos_core/cherry-picked-1110-beta-into-dev/docs/source/modules/zos_backup_restore.rst

File: docs/source/modules/zos_backup_restore.rst (hunk 14 above)
Comment (ddimatos) [reply docs/source/modules/zos_backup_restore.rst]: I suspect the correction removes a diff so the file does not show up in the PR but is in in the branch as it should be so I am resolving this.
https://raw.githubusercontent.com/ansible-collections/ibm_zos_core/cherry-picked-1110-beta-into-dev/docs/source/modules/zos_backup_restore.rst

**Review by ddimatos — APPROVED** [review]
Looks good to me, thanks for working on the edits. Nice work @fernandofloresg

**Review by richp405 — APPROVED** [review]
Thanks... that was a BOATLOAD of changes... looks much better now.

**Pull request merged**